common/psbt_internal.c

```diff
@@ -101,3 +101,32 @@ psbt_to_witness_stacks(const tal_t *ctx,
 	tal_resize(&stacks, stack_index);
 	return stacks;
 }
+
+void psbt_add_keypath_to_last_output(struct bitcoin_tx *tx,
```

Comment:
this is somewhat similar to `wallet_set_keypath`.  can these be DRYed?

Reply:
done, moved to `psbt_keypath.[hc]`